public/assets/main.css

```diff
@@ -12,6 +12,9 @@ a.game__nav:focus {
 button {
   margin: 0.5rem 0.5rem 0 0;
 }
+button + button {
```

Comment:
does this mess up any other views?

Reply:
**If** there were buttons next to each other (without any el between them!) that aren’t inline and don’t use inline-styles for margin, maybe. I haven’t found any.
To me this is a useful default, hence I put it here.